docs/root/version_history/current.rst

```diff
@@ -27,6 +27,12 @@ Incompatible Behavior Changes
   vendor the corresponding protobuf definitions to ensure that the
   renumbered fields have the types expected by those releases.
 * ext_authz: fixed skipping authentication when returning either a direct response or a redirect. This behavior can be temporarily reverted by setting the ``envoy.reloadable_features.http_ext_authz_do_not_skip_direct_response_and_redirect`` runtime guard to false.
+* extensions: deprecated extension names now default to triggering a configuration error.
+  The previous warning-only behavior can be reverted by setting the runtime key
```

Comment:
can be _temporarily_ reverted
here and below.

Reply:
I'm actually not sure where the grpc_bridge_filter and tracing message came from, but I'll fix my entry up.

Reply:
Turns out I must have accidentally copied them from v1.19.0 notes. I've removed them since they're not on master and I haven't changed anything related to them.